attestation-agent/attester/src/se/seattest.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) Copyright IBM Corp. 2024
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::*;
+use async_trait;
+
+#[derive(Default)]
+pub struct FakeSeAttest {}
+
+#[async_trait::async_trait]
+pub trait SeImplAttester {
```

Comment:
This trait doesn't seem necessary to me. Why not just put the logic in the methods above (and call into the real SE crate from there once it is ready)? Also, `SeImplAttester` is a slightly strange name for a trait.

Reply:
Thanks @fitzthum , we'll put the real crate in https://crates.io/crates/s390_pv, it's still in-progress. So, `FakeSeAttest` and `SeImplAttester` will be replaced the crate eventually.

Reply:
Fixed.